tbcrawler/pytbcrawler.py

```diff
@@ -55,6 +57,11 @@ def run():
     job_config = ut.get_dict_subconfig(config, args.config, "job")
     job = crawler_mod.CrawlJob(job_config, url_list)
 
+    # Run display
+    xvfb_wrapper = XvfbWrapper()
+    xvfb_wrapper._from(args.virtual_display)
+    xvfb_wrapper.start()
+
```

Comment:
Why not do input checking in a function?
I think we don't need a class for this:
You need https://github.com/webfp/tor-browser-selenium/blob/d3e8eef55310aced5c5bf8ae92061e4a6c3120c5/tbselenium/utils.py#L47-L56
and some input checking, no?